### ISSUE TITLE
Does not work in Firefox

### ISSUE THREAD

**Issue by J4mez** [fireship-io/next13-pocketbase-demo#7]
In Firefox, when creating a new note the site crashes and an unhanded network fetch error is displayed for a split second. 
Tested same instance in edge and chrome witch both worked fine.

**Comment by kellenproctor** [comment]
Same problem for me. At this point, this is probably a next 13 bug.

**Comment by DevinCLane** [comment]
Yep I had the same issue on Firefox, works on Chrome. Not sure what the issue is.

**Comment by Alex0White** [comment]
![image](https://user-images.githubusercontent.com/26420357/200165783-ceff82d8-abdb-46d4-81a2-0078ba60187d.png)
![image](https://user-images.githubusercontent.com/26420357/200165839-2afd4cc3-c1a4-4436-9221-cde6e9db2139.png)

**Comment by chintogtokh** [comment]
I'm late to the party but looks like you need to add a `e.preventDefault()` in the [`create` function here](https://github.com/fireship-io/next13-pocketbase-demo/blob/main/app/notes/CreateNote.tsx). Then need to manually refresh the page post-request, like using `router.refresh()`.

I think its just due to how Firefox/Chrome handle form requests differently.
